openpype/settings/defaults/project_anatomy/templates.json

```diff
@@ -38,16 +38,6 @@
             "file": "{subset}_{@version}<_{output}><.{@frame}>.{ext}",
             "path": "{@folder}/{@file}"
         },
-        "simpleUnrealTextureHero": {
-            "folder": "{root[work]}/{project[name]}/{hierarchy}/{asset}/publish/{family}/hero",
-            "file": "{originalBasename}.{ext}",
-            "path": "{@folder}/{@file}"
-        },
-        "simpleUnrealTexture": {
-            "folder": "{root[work]}/{project[name]}/{hierarchy}/{asset}/publish/{family}/{@version}",
-            "file": "{originalBasename}_{@version}.{ext}",
-            "path": "{@folder}/{@file}"
-        },
```

Comment:
Were these unused?

Reply:
There seems to be quite some more usage of "simpleUnrealTexture" in the code base?

Reply:
True, this PR #6012 should eradicate it completely